src/mail/editor/SendMailModel.ts

```diff
@@ -333,6 +331,10 @@ export class SendMailModel {
 		// if we reuse the same image references, changing the displayed mail in mail view will cause the minimized draft to lose
 		// that reference, because it will be revoked
 		this.loadedInlineImages = cloneInlineImages(inlineImages)
+
+		// We need to know if the user is replying from email view to respect its decision about external contents
+		this.isReplyingFromEmailView = true
```

Comment:
Since it's a condition used by the editor and **doesn't need** to be stored into the email object, makes sense to move to the ```MailEditor``` as an optional parameter with default value set to false.